image/templates/helm/stackrox-secured-cluster/templates/collector-config.yaml

```
@@ -0,0 +1,17 @@
+{{- include "srox.init" . -}}
+
+{{- if ._rox.collector.runtimeConfig.enable -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: collector-config
+  namespace: {{ ._rox._namespace }}
+  labels:
+    {{- include "srox.labels" (list . "configmap" "collector-config") | nindent 4 }}
+    auto-upgrade.stackrox.io/component: "sensor"
+  annotations:
+    {{- include "srox.annotations" (list . "configmap" "collector-config") | nindent 4 }}
+data:
+  runtime_config.yaml: |
+    {{- tpl (.Files.Get "config-templates/collector/config.yaml.tpl") . | nindent 4 }}
+{{ end -}}
```

```suggestion
{{- end }}
```

(See https://github.com/stackrox/stackrox/blob/056a7cf4683fae31889bf17636b89e47976f571c/image/templates/CHANGING_CHARTS.md#L5)